x-pack/test_serverless/functional/test_suites/security/config.feature_flags.ts

```diff
@@ -18,7 +18,7 @@ export default createTestConfig({
   },
   suiteTags: { exclude: ['skipSvlSec'] },
   // add feature flags
-  kbnServerArgs: [],
+  kbnServerArgs: ['--xpack.security.roleManagementEnabled=true'],
```

Comment:
__question:__ same question here, should be enable it for all project types to make sure it works the same everywhere?

Reply:
++ will add to all 3

Reply:
Fixed in latest commit